src/dotnet/APIView/APIViewWeb/DTOs/SiteNotificationDto.cs

```diff
@@ -1,44 +1,91 @@
-using System;
-using APIViewWeb.Models;
+using System.Text.Json.Serialization;
 
-namespace APIViewWeb.DTOs;
-
-public class SiteNotificationDto
+namespace APIViewWeb.DTOs
 {
-    public string ReviewId { get; set; }
-    public string RevisionId { get; set; }
-    public string Title { get; set; }
-    public string Summary { get; set; }
-    public string Message { get; set; }
-
-    private string _status;
-    public string Status
+    public class SiteNotificationDto
```

Comment:
I think you can just make this a Model, especially since it has a private method with added logic. Use DTo when you are purely just transferring data.